answer/lib.rs

```diff
@@ -19,6 +19,7 @@ use encoding::{
     value::value::Value,
     AsBytes,
 };
+use encoding::graph::type_::vertex::TypeVertexEncoding;
```

Comment:
I moved `fn vertex(&self)` into TypeVertexEncoding. We already use `encoding` in many places in `concept` even if we were interested to reduce the amount of its usages on the higher logic's level, so it doesn't affect much.
It allows me to have both `into_vertex` and `vertex` (shall I rename every `fn vertex(&self)` to `fn as_vertex(&self)`?..) in the same trait.